aineko/cli/dream.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""A wrapper around API calls to the Aineko Dream API."""
+
+import click
+import requests
+
+
+@click.command()
+@click.argument("prompt")
+@click.argument("api-key")
+@click.option(
+    "-u",
+    "--url",
+    default="https://dream.ap-2359264f-9cba-43ba-9ac4-ae3ba7b1d4b1.aineko.app",
+    help="API url to use for the Aineko Dream API.",
+)
+def dream(prompt: str, api_key: str, url: str) -> None:
+    """Main function generate an aineko project using Aineko Dream.\f
+
+    Args:
+        prompt: Prompt to generate a project from.
+        api_key: API key to use for the Aineko Dream API.
+        url: URL to use for the Aineko Dream API.
+    """
+    try:
+        click.echo("Generating project from prompt...")
+
+        r = requests.get(
+            url=f"{url}/create/",
+            params={"prompt": prompt},
+            headers={"x-api-key": api_key},
+            timeout=300,
```

Comment:
Might want to make this a config, also is this set to 300 for any reason? Is this based on the timeout for GPT-4?

Reply:
timeout is based on our load balancer timeouts